scheduler/src/cook/mesos/api.clj

```diff
@@ -815,8 +815,8 @@
   (->> datasets
        (map (fn [{:keys [dataset partitions]}]
               (cond-> {:dataset (walk/stringify-keys dataset)}
-                partitions
-                (assoc :partitions (into #{} (walk/stringify-keys partitions))))))
+                      partitions
```

Comment:
`cond->` conditions and clauses should be indented one level. You can change Cursive settings to fix this.

Reply:
Reverted.